docker-compose-lint.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  avalon-lint:
+    container_name: avalon-lint
```

Comment:
Remove container_name if you want to run it as part of CI

Reply:
Thanks @pankajgoyal2, nice catch. Fixed.
Tested with error case as well @manojgop. Does fail the Jenkins build.